cocos/ui/UIEditBox/UIEditBox.h

```diff
@@ -407,7 +407,68 @@ namespace ui {
          * @param returnType One of the EditBox::KeyboardReturnType constants.
          */
         void setReturnType(EditBox::KeyboardReturnType returnType);
-            
+        
+        /**
+         * Set the inner padding in edit box.
+         * @param padding left, top, right, bottom.
```

Comment:
The doc string here is not confirm to the rule of Doxygen.
It should be 
```
@param left   The left padding.
@param top The top padding.
@param right The right padding.
@param bottom The bottom padding.
```

Only with this format we could generate the correct developer reference.